test/integration/targets/azure_rm_networkinterface/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+- name: Create virtual network
```

Comment:
It's great that you have some tests, but it would be better to also test out each operation under check mode, a normal run and finally an idempotent run.